cmd/grill/main.go

```diff
@@ -6,7 +6,7 @@ import (
 	"log"
 	"os"
 
-	"github.com/echlebek/grill/grill"
+	grill "github.com/echlebek/grill/internal"
```

Comment:
Better to have a subdirectory in internal called grill, internal protects the entire subtree from being imported.

Reply:
PTAL